common/views/components/DefaultPageLayout/DefaultPageLayout.js

```diff
@@ -82,54 +147,84 @@ const Analytics = ({
   featureFlags = [],
   isPreview = false
-}) => (
+}: Props) => (
 
   <div>
     <Head>
-      <meta charSet="utf-8" />
+      <meta charSet='utf-8' />
       {/* TODO: use flag as to whether to include this */}
       <Analytics
         category={null}
         seriesUrl={null}
         positionInSeries={null}
         contentType={null} />
 
-      <meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
-      <title>{title && `${title} | `}Wellcome Collection</title>
-      <meta name="viewport" content="width=device-width, initial-scale=1" />
-      <meta name="theme-color" content="#000000"/>
-      {pageMeta}
+      <meta httpEquiv='X-UA-Compatible' content='IE=edge,chrome=1' />
+      <title>{`${title} | Wellcome Collection`}</title>
+      <meta name='viewport' content='width=device-width, initial-scale=1' />
+      <meta name='theme-color' content='#000000'/>
+
+      <OpenGraph
+        type={type}
+        url={url}
+        title={title}
+        description={description}
+        imageUrl={imageUrl}
+      />
+      <TwitterCard
+        type={type}
+        url={url}
+        title={title}
+        description={description}
+        imageUrl={imageUrl} />
 
       {/* CSS */}
 
-      <link rel='preload' href='/static/css/non-critical.css' as="style" onLoad='this.rel="stylesheet"' />
+      <link rel='preload' href='/static/css/non-critical.css' as='style' onLoad='this.rel="stylesheet"' />
       <link rel='apple-touch-icon' sizes='180x180' href='/static/icons/apple-touch-icon.png' />
       <link rel='shortcut icon' href='/static/icons/favicon.ico' type='image/ico' />
-      <link rel='icon' type='image/png' href='/static/icons/favicon-32x32.png' sizes="32x32" />
-      <link rel='icon' type='image/png' href='/static/icons/favicon-16x16.png' sizes="16x16" />
+      <link rel='icon' type='image/png' href='/static/icons/favicon-32x32.png' sizes='32x32' />
+      <link rel='icon' type='image/png' href='/static/icons/favicon-16x16.png' sizes='16x16' />
       <link rel='manifest' href='/static/icons/manifest.json' />
       <link rel='mask-icon' href='/static/icons/safari-pinned-tab.svg' color='#000000' />
       <script src='/static/libs/picturefill.min.js' async />
       {/* Leaving this out for now as it's hanging locally for me */}
       {/* <script src='//platform.twitter.com/widgets.js' async defer></script> */}
       <HeadJs enhancedJsPath='/static/js/app.js' />
-      <script type="application/ld+json">{/* JSON+LD Z */}</script>
+      <script type='application/ld+json'>{/* JSON+LD Z */}</script>
       <script dangerouslySetInnerHTML={{ __html: `
       window.WC = {
         featuresCohort: ${JSON.stringify(featuresCohort)},
         featureFlags: ${JSON.stringify(featureFlags)}
       }
     `}} />
-      {canonicalUri && <link rel="canonical" href={canonicalUri} />}
+      {url && <link rel='canonical' href={url} />}
     </Head>
 
     <div className={isPreview ? 'is-preview' : undefined}>
```

Comment:
Is this ternary required, or would JSX just leave out the class altogether with a falsy `className={isPreview}`?

Reply:
Yea, but how would it know to use the string `is-preview`?

We could create something that takes an `{[string]: boolean}` but haven't found many cases for it it.

e.g.
```jsx
<div className={classnames({ 'is-preview' : isPreview })}></div>
```

Reply:
Of course – I'm being slow.